realm/realm-library/src/objectServer/java/io/realm/SyncConfiguration.java

```diff
@@ -314,15 +317,49 @@ private void validateAndSet(String uri) {
             }
 
             try {
-                serverUrl = new URI(uri);
+                serverUrl = new URI(uri.toLowerCase());
```

Comment:
why `toLowerCase()`? URI should be case sensitive, at least I guess Apache & nginx are